lerobot/templates/visualize_dataset_template.html

```diff
@@ -2,217 +2,334 @@
         </div>
-        <p id="graph-timer" class="font-mono ml-14 mt-4"></p>
-      </div>
-      <div id="labels" class="columns-2" style="font-size: 0rem;">
-      </div>
     </div>
-  </div>
-
-  <script>
-    document.addEventListener('DOMContentLoaded', function () {
-      const videos = document.querySelectorAll('video');
-      const video = videos[0];
-      const timer = document.getElementById('timer');
-
-      // controllers
-      const slider = document.getElementById('slider');
-      const btnPlay = document.getElementById('btn-play');
-      const btnPause = document.getElementById('btn-pause');
-      const btnBackward = document.getElementById('btn-backward');
-      const btnForward = document.getElementById('btn-forward');
-      const btnRewind = document.getElementById('btn-rewind');
-
-      const generateDarkModeColor = () => {
-        // Function to generate a random number within a given range
-        const randomInt = (min, max) => Math.floor(Math.random() * (max - min + 1)) + min;
-        // Generate hue, saturation, and lightness
-        const hue = randomInt(0, 360); // Hue from 0 to 360
-        const saturation = randomInt(60, 100); // High saturation for vivid colors
-        const lightness = randomInt(50, 75); // Lightness adjusted for visibility on dark backgrounds
-        // Return the HSL color string
-        return `hsl(${hue}, ${saturation}%, ${lightness}%)`;
-      }
 
-      let dygraphTime = 0.0;
-      const dygraph = new Dygraph(document.getElementById('graph'), '{{ ep_csv_url }}', {
-        pixelsPerPoint: 0.01,
-        legend: 'always',
-        labelsDiv: document.getElementById('labels'),
-        labelsKMB: true,
-        colors: Array.from({
-          length: 100
-        }, generateDarkModeColor), // custom color palette
-        pointClickCallback: (event, point) => (dygraphTime = point.xval),
-        highlightCallback: (event, x, points, row, seriesName) => (dygraphTime = x),
-      });
-      setInterval(() => (document.getElementById("graph-timer").innerText = `Time: ${dygraphTime.toFixed(3)}s`),
-        200); // keep "graph-timer" in sync
-
-      video.addEventListener('loadedmetadata', () => (timer.textContent = formatTime(0) + ' / ' + formatTime(video
-        .duration)));
-      video.addEventListener('timeupdate', () => (timer.textContent = formatTime(video.currentTime) + ' / ' +
-        formatTime(video.duration)));
-      const formatTime = (time) => {
-        var hours = Math.floor(time / 3600);
-        var minutes = Math.floor((time % 3600) / 60);
-        var seconds = Math.floor(time % 60);
-        return (hours > 0 ? hours + ':' : '') + (minutes < 10 ? '0' + minutes : minutes) + ':' + (seconds < 10 ?
-          '0' + seconds : seconds);
-      }
+    <script>
+        function createAlpineData() {
+            return {
+                dygraph: null,
+                currentFrameData: null,
+                columnNames: ["state", "action", "pred action"],
```

Comment:
Not a blocker for this PR, but I am wondering how we could add extra values like the loss, loss_l1, (potentially reward, and other plots):

<img width="801" alt="Screenshot 2024-07-30 at 11 32 46" src="https://github.com/user-attachments/assets/3cf7da43-9071-42ae-9fd8-db2ae8cd7456">

If you rebase on top of my branch and add `--policy-method forward` you will be able to save the loss and loss_l1

Reply:
https://github.com/huggingface/lerobot/pull/346/commits/c8d647d2be62398e8d365405f4c4def8548703ad no more dotted lines. All lines are full lines now without distinction

Reply:
yeah happy to extend the columns behaviour in the follow up PR